files/galaxy/dynamic_rules/usegalaxy/total_perspective_vortex/tools.yml

```diff
@@ -621,7 +621,7 @@ tools:
     cores: 1
     mem: 6
 
-  toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MarkDuplicates/*:
+  toolshed.g2.bx.psu.edu/repos/devteam/picard/picard_MarkDuplicates/.*:
     cores: 8
     mem: 20
```

Comment:
Is there a reason for not using the memory definition in
https://github.com/galaxyproject/tpv-shared-database/blob/fe3ba41340dac17de585738c6a39539ca776c8b1/tools.yml#L495 ?
Is this extra-overhead for docker?